AppCenterData/AppCenterData/MSDataErrors.h

```diff
@@ -38,5 +39,6 @@ static NSString const *kMSACDataInvalidClassDesc = @"Provided class does not con
 static NSString const *kMSACDataCosmosDbErrorResponseDesc = @"Unexpected error while talking to CosmosDB.";
 static NSString const *kMSACDocumentCreationDesc = @"Can't create document.";
 static NSString const *kMSACDataErrorDocumentIdInvalidDesc = @"Invalid document ID.";
+static NSString const *kMSACDataErrorNextDocumentPageUnavailable = @"Can't retrieve next page when the device is offline.";
```

Comment:
We're not quite retrieving the next page, perhaps "detect" instead of "retrieve"?

Also to align with other error messages and add `Desc` suffix.

```suggestion
static NSString const *kMSACDataErrorNextDocumentPageUnavailableDesc = @"Can't detect next page when the device is offline.";
```